shops/management/commands/import_weekly_data.py

```diff
@@ -19,7 +19,9 @@ def add_arguments(selef, parser):
 
     def handle(self, *args, **kwargs):
         delete_data = input('You are about to delete all existing weekly data, are you sure? (YES/Y to confirm, any other key to quit)')
-        if delete_data.lower() in ['y', 'yes']:
+        if delete_data.lower() not in ['y', 'yes']:
+            pass
```

Comment:
If you replace 'pass' with 'return', then you can drop the 'else' and drop the indentation for the rest of the code in this function